sbncode/SinglePhotonAnalysis/SEAview/seaDBSCAN.h

```diff
@@ -0,0 +1,159 @@
+/**
+ * \file seaDBSCAN.h
```

Comment:
How does this differ from DBSCAN? If they don't differ, or there's only some small difference that can be abstracted, then they should be merged.